target/linux/ath79/dts/qca9558_tplink_rex5x.dtsi

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		mdio-gpio0 = &mdio2;
```

Comment:
Hmm, could you explain why you need to create an alias for your mdio interface here? I know two other repeaters do this, however i see nothing taking advantage of this alias so it should be safe to drop.

In this case, the `mdio2` handle should also be dropped from your `mdio` node.

Reply:
The reason is, that i took the re450v2 device tree as a template, saw it, thought the same thing and found [this](https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/net/mdio-gpio.txt#L11).
If you think it should be removed, I'll do so. But then we should clean up re350k-v1, re450-v2 and a bunch of ipq806x as well.

Reply:
Didn't know that. In this case it is fine.